Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,243 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct BrowseOGCAPIFeatureServiceView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// A Boolean value indicating whether the alert is presented.
+    @State private var alertIsPresented = true
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The input obtained from the user for the OGC service URL.
+    @State private var featureServiceURL = "https://demo.ldproxy.net/daraa"
+    
+    /// The selected layer name.
+    @State private var selection: String = ""
+    
+    var body: some View {
+        MapViewReader { mapProxy in
+            MapView(map: model.map)
+                .toolbar {
+                    ToolbarItemGroup(placement: .bottomBar) {
+                        // The button in toolbar that allows user to launch the load alert.
+                        Button("Open Service") {
+                            alertIsPresented = true
+                        }
+                        Spacer()
+                        // The layers button will not appear until selection is set.
+                        if !selection.isEmpty {
+                            Picker("Layers", selection: $selection) {
+                                ForEach(model.layerNames, id: \.self) { title in
+                                    Text(title)
+                                }
+                            }
+                            .task(id: selection) {
+                                model.update(for: selection)
+                                if let selection = model.selectedInfo {
+                                    do {
+                                        try await model.displayLayer(with: selection)
+                                        if let extent = model.completeExtent {
+                                            await mapProxy.setViewpointGeometry(
+                                                extent,
+                                                padding: 100
+                                            )
+                                        }
+                                    } catch {
+                                        self.error = error
+                                    }
+                                }
+                            }
+                            .pickerStyle(.automatic)
+                        }
+                    }
+                }
+                .alert("Load OGC API feature service", isPresented: $alertIsPresented, actions: {
+                    // Alert is set with a default url for the OGC API, the user can update the url.
```

Comment:
```suggestion
```
I'm not sure this comment is necessary. If so, it could be reworded to avoid a run-on sentence.